src/MagicScaler/Experimental/TransformFactory.cs

```diff
@@ -10,6 +10,20 @@ namespace PhotoSauce.MagicScaler.Experimental;
 /// <summary>Contains methods for creating <see cref="IPixelSource" /> transforms. These may be changed or removed in future releases.</summary>
 public static class TransformFactory
 {
+	/// <summary>Create a processor that converts raw pixel data to the given pixel format, optionally using the given ICC color profiles for gamma correction.</summary>
+	/// <param name="sourceFormat">The binary representation of the source pixel data.  Must be one of the values from <see cref="PixelFormats"/>.</param>
+	/// <param name="destFormat">The binary representation of the destination pixel data.  Must be one of the values from <see cref="PixelFormats"/>.</param>
+	/// <param name="sourceProfile">The ICC color profile to use for the source pixel data. If null, sRGB will be used.</param>
+	/// <param name="destProfile">The ICC color profile to use for the resulting pixel data. If null, sRGB will be used.</param>
+	/// <returns>An <see cref="IPixelProcessor"/> that can be used to process pixel data.</returns>
+	[CLSCompliant(false)]
```

Comment:
ugh. I forgot putting `[CLSCompliant]` on the interface would make it required on the method returning the interface as well.  Better to move it to the method in the interface so it can be removed from here.

Reply:
![sideshow-bob-rake](https://github.com/user-attachments/assets/6e03f874-1ef3-47be-97c3-91d03433131c)